src/Analysis/GWASResults/Views/Home/HomeTable/ActionsDropdown/ActionsDropdown.jsx

```diff
@@ -3,8 +3,11 @@ import PropTypes from 'prop-types';
 import { Space, Dropdown, Button } from 'antd';
 import { EllipsisOutlined } from '@ant-design/icons';
 import {
-  fetchPresignedUrlForWorkflowArtifact,
+  fetchPresignedUrlForWorkflowArtifact, retryWorkflow,
 } from '../../../../Utils/gwasWorkflowApi';
+import PHASES from '../../../../Utils/PhasesEnumeration';
+
+/* eslint no-alert: 0 */ // --> OFF
```

Comment:
Please remove this comment if we can get the AntD notification to work.

Reply:
done